### PR TITLE
`qscout` compile

### PR DESCRIPTION
**Pull request by vtomole**

### CODE REVIEW

**Comment by singular-value** [comment]
Include integration test?

File: examples/qscout_compile.ipynb [hunk 1]
```diff
@@ -0,0 +1,272 @@
+{
```
Comment (singular-value) [inline examples/qscout_compile.ipynb:1]: <p>QSCOUT</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/cirq-superstaq/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Review by singular-value — CHANGES_REQUESTED** [review]
- Looks like this PR extraneously adds a new file, finance_ouput.ipynb

**Comment by vtomole** [comment]
> Looks like this PR extraneously adds a new file, finance_ouput.ipynb
Removed.

File: examples/qscout_compile.ipynb (hunk 1 above)
Comment (vtomole) [reply examples/qscout_compile.ipynb:1]: Done.

**Review by singular-value — COMMENTED** [review]
no other comments from my end

**Review by singular-value — COMMENTED** [review]
lgtm

**Comment by vtomole** [comment]
@singular-value Will need official approval to merge.

**Review by singular-value — APPROVED** [review]

**Pull request merged**